src/clj/game/cards/events.clj

```diff
@@ -809,6 +820,10 @@
       :effect (req (show-wait-prompt state :corp "Runner to rearrange the top cards of their stack")
                    (let [from (take 6 (:deck runner))]
                      (continue-ability state side (entrance-trash from) card nil)))})
+   
+   "Mars for Martians"
+   {:msg (msg "draw " (count (filter #(and (has-subtype? % "Clan") (is-type? % "Resource")) (all-installed state :runner))) " card(s) and gain " (:tag runner) " [Credits]")
```

Comment:
Can streamline by starting with: 

`(let [n (count (filter #(and (has-subtype? % "Clan") (is-type? % "Resource")) (all-installed state :runner)))]`